backend/api/admin_views.py

```
@@ -151,4 +151,28 @@ def retrieve(self, request, *args, **kwargs):
             'most_viewed': most_viewed_serializer.data,
             'most_liked': most_liked_serializer.data,
             'recent': recent_serializer.data
-        })
\ No newline at end of file
+        })
+
+class WebcamRecordingsView(generics.ListAPIView):
+    permission_classes = [IsAuthenticated, IsAdmin]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)